constellations/gemini-admin/src/help/help.sass

```diff
@@ -1,24 +1,24 @@
 @import "../_.constants.sass"
 
-body
-  margin: 0
+#view-help
```

Comment:
As for formatting, we should try to keep the indents 4 spaces (not 2 which you see in this file). Which code editor are you using? With VSCode there's a way to format your sass files

Reply:
Yeah i'm using VSCode